console/src/components/Services/Events/EventTriggers/Add/Add.tsx

```diff
@@ -360,7 +360,7 @@ type PropsFromState = {
 
 const mapStateToProps: MapStateToProps<PropsFromState> = state => {
```

Comment:
@beerose So we do have  a type defined on the `ReduxState` as can be seen below. 

But the issue is that since it can be one of two possible types we are doing the assertion. I could not see sufficient overlap between `MetadataObject` and `Table` hence I could not merge them into a single type.


https://github.com/hasura/graphql-engine/pull/5028/files#diff-e139d4c1a6a1506f625a3d0d82f8be06R20